index.html

```diff
@@ -2,13 +2,40 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Formular</title>
+    <link href="style.css" rel="stylesheet">
 </head>
 
 <body>
+    <nav class="nav">
+        <ul>
+            <li> <a href="https://www.smashingmagazine.com/" target="_blank" rel="noopener noreferrer"> <img width="30"
+                        src="https://cdn.schema.io/smashing-magazine/62267a6a93cebd01324e4509/aa48d1a46ce4ca09c1962d9e939e678c"
+                        alt="logo">
+                </a> </li>
+
+            <li> <a href="https://www.smashingmagazine.com/articles/" target="_blank" rel="noopener noreferrer">
+                    Articles </a></li>
+
+            <li> <a href="https://www.smashingmagazine.com/printed-books/"> Books </a></li>
+            <li> <a href="https://www.smashingmagazine.com/online-workshops/" target="_blank" rel="noopener noreferrer">
+                    Workshops </a> </li>
+            <li> <a href="https://www.smashingmagazine.com/events/" target="_blank" rel="noopener noreferrer">
+                    Conferences </a> </li>
+            <li> <a href="https://www.smashingmagazine.com/membership/" target="_blank" rel="noopener noreferrer">
+                    Membership </a> </li>
+        </ul>
+        <form>
+
+            <input placeholder=" Search articles..." type="text" name="search">
```

Comment:
Why is there an empty space at the beginning of this placeholder? Let's remove it.